piwik/content.md

```diff
@@ -0,0 +1,71 @@
+# Piwik
+
+[![Build Status](https://travis-ci.org/piwik/docker-piwik.svg?branch=master)](https://travis-ci.org/piwik/docker-piwik)
+
+Piwik is the leading open-source analytics platform that gives you more than just powerful analytics:
+ - Free open-source software
+ - 100% data ownership
+ - User privacy protection
+ - User-centric insights
+ - Customisable and extensible
+
+%%LOGO%%
+
+# How to use this image
+
+The easiest is to use our `docker-compose.yml`.
+
+Make sure you have [docker-compose](http://docs.docker.com/compose/install/) installed. And then:
+
+```
+git clone https://github.com/piwik/docker-piwik.git
```

Comment:
I have also had trouble getting the compose file to work out of the box.  The ``.env`` file only has one key listed for ``MYSQL_ROOT_PASSWORD``, which could be placed directly into the compose file.  Please carefully review the compose file as I suspect there are some unnecessary things going on.